terraform/modules/aws_cargohold-asset-storage/outputs.tf

```diff
@@ -0,0 +1,18 @@
+# Output required to configure wire-server
+
+output "bucket_name" {
+  value = aws_s3_bucket.asset_storage.bucket
+}
+
+output "s3_endpoint" {
+  value = "https://s3.${var.region}.amazonaws.com"
+}
+
+output "cargohold_access_key" {
+  value = aws_iam_access_key.cargohold.id
+}
+
+# This value is sensitive in nature and you cannot get it from AWS later.
+output "cargohold_access_secret" {
+  value = aws_iam_access_key.cargohold.secret
+}
```

Comment:
As I commented on the previous PR on the other repo, this is not sustainable (see inline comment around the same code there).

Reply:
Since it is a TF output, it is stored in the `.tfstate` and can be queried at all times with `terraform output`. So the comment is correct but also misleading. I adjusted it. But I think it is not faesible to put the comment on all outputs in any module which expose sensitive data

Reply:
never mind. saw [your comment](https://github.com/wireapp/wire-server-deploy/pull/244#discussion_r416607447)